content/doc/book/managing/cli.adoc

```diff
@@ -27,24 +27,18 @@ be convenient for scripting of routine tasks, bulk updates, troubleshooting,
 and more.
 
 The command line interface can be accessed over SSH or with the Jenkins CLI
-client, a `.jar` file distributed with Jenkins. The SSH approach is preferred
-over the CLI client as it is considered more secure.
+client, a `.jar` file distributed with Jenkins.
 
-////
-Background:
-23:19 <@rtyler> danielbeck: I'm writing CLI documentation, should I just encourage people to use SSH or stick with the default of jenkins-clir.jar?
-23:20 <@rtyler> we've told enough people to turn off the CLI over the past year...
-23:20 <@rtyler> :/
-23:21 <@danielbeck> it's probably better in general, but e.g. firewalls rules may require regular CLI use
-23:21 <@danielbeck> in any case, auth needs to use SSH keys
-////
+Prior to Jenkins 2.54, use of the custom CLI client is not recommended for security reasons:
```

Comment:
Please do not conflate JNLP with the CLI protocol.

They are two different protocols, and JNLP4 should now be just as safe as SSH for launching a build agent. 

The issue with the CLI is fundamentally exposing the remoting protocol *with the serialisation vectors* to persons other than the Jenkins Admin.

We need admins to launch agents either with JNLP4 or SSH and conflating with the CLI protocol may cause confusion and drive them to SSH as the only slave launch mechanism (which is bad)